config/settings.py

```diff
@@ -92,6 +92,7 @@
     "hexa.user_management.middlewares.TwoFactorMiddleware",
     "hexa.plugins.connector_airflow.middlewares.dag_run_authentication_middleware",
     "hexa.pipelines.middlewares.pipeline_run_authentication_middleware",
+    "hexa.workspaces.middlewares.workspace_authentication_middleware",
```

Comment:
Maybe we could be more explicit and call it `workspace_token_authentication_middleware`